### PR TITLE
Linear Actuator Files

### LINKED ISSUE #78: Class to use linear actuator over serial
## Description
For placing the module we will be using a linear actuator (similar to a piston in Minecraft). We can control it using a serial connection, so we need to create a class with accessors like extend (which should go to a set percent extension) as well as being able to read its current status.

https://github.com/RoverRobotics/liblac
https://github.com/DamnedFacts/actuonix-lac

## Requirements
- [ ] Figure out how to interact with LA
- [ ] Compose class with extend, read, and any other useful methods
- [ ] Complete functionality of member methods
- [ ] Do above in own module/files

### PR DESCRIPTION
**Pull request by SercanAkcay**
Closes #78

### CODE REVIEW

File: flight/utils/Wrapped_LAC.py [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+    self.piston.set_speed(maxSpeed)
+    print("Piston set to max speed")
+    
+    #Ensures the LAC will not hit mechanical stops
+    if (rPosition <= rMechStop):
+      return False
+    elif (ePosition >= eMechStop):
+      return False
+    else:
+      return True
```
Comment (ClayMav) [inline flight/utils/Wrapped_LAC.py]: Move the checks for mechstop to the init function, and return nothing from this function

File: flight/utils/Wrapped_LAC.py [hunk 2]
```diff
@@ -0,0 +1,137 @@
+#!/usr/bin/python
+import asyncio
+from lac import *
```
Comment (ClayMav) [inline flight/utils/Wrapped_LAC.py]: don't use `import *` just import each of the individual elements used

File: flight/utils/Wrapped_LAC.py [hunk 3]
```diff
@@ -0,0 +1,137 @@
+#!/usr/bin/python
+import asyncio
```
Comment (ClayMav) [inline flight/utils/Wrapped_LAC.py]: is this used

File: flight/utils/Wrapped_LAC.py [hunk 4]
```diff
@@ -0,0 +1,137 @@
+#!/usr/bin/python
```
Comment (ClayMav) [inline flight/utils/Wrapped_LAC.py]: this should never be called so remove the shebang

File: flight/utils/Wrapped_LAC.py [hunk 5]
```diff
@@ -0,0 +1,137 @@
...
+import asyncio
+from lac import *
+from mavsdk import System
```
Comment (ClayMav) [inline flight/utils/Wrapped_LAC.py]: Never used

**Review by ClayMav — CHANGES_REQUESTED** [review]
Can't approve until i run it on my machine to test it.

**Comment by SercanAkcay** [comment]
Got it I’ll make the changes

File: flight/utils/Wrapped_LAC.py [hunk 6]
```diff
@@ -0,0 +1,135 @@
...
+class sLAC:
+  def __init__(self):
+    self.piston = LAC(vendorID,productID)
+    self.setupLAC()
+    
+    #Ensures the LAC will not hit mechanical stops
+    if (rPosition <= rMechStop):
+      raise Exception("Retract is set to the mechanical stop.")
+    elif (ePosition >= eMechStop):
+      raise Exception("Extend is set to the mechanical stop.")
```
Comment (ClayMav) [inline flight/utils/Wrapped_LAC.py:65]: move these above everything else in the function

**Review by ClayMav — CHANGES_REQUESTED** [review]

**Review by ClayMav — APPROVED** [review]
codacy has a few issues you should fix but I am satisfied once she is

**Pull request merged**